spec/models/manageiq/providers/redhat/infra_manager/refresher_3_0_spec.rb

```diff
@@ -363,7 +363,7 @@ def assert_specific_vm_powered_on
       :ems_ref_obj   => nil,
       :uid_ems       => "eb5592d2-ce76-11e0-a703-005056af0085_vm",
       :name          => "vm",
-      :is_datacenter => false,
+      :type          => nil,
```

Comment:
Type should never be nil in an STI situation.  At the very least it should be the base model, which is `"EmsFolder"`...this is why I think you need a data migration for the type column,.